### DISCUSSION TITLE
How to zoom in further

### DISCUSSION THREAD

**Discussion by simonfruehauf** [javalent/obsidian-leaflet#252] [Q&A] [answered]
I've got an SVG image that I'm using as a map, and I'd like to be able to zoom in further than I am able to right now. However, I'm not able to achieve this with the `maxZoom` and `minZoom` parameters, apart from setting min higher than max (which permanently zooms in and doesn't let me change the zoom level anymore). 

Decimal places (e.g. setting minZoom to 0.1) don't seem to work either.

My current setup is as follows, but I'd like to zoom in further (technically even infinitely, since it is an SVG after all). 

```leaflet
id: continent-map
image: [[Image.svg]]]
height: 650px
maxZoom: 2
minZoom: 1
unit: kilometers
zoomDelta: 0.2
```

**Comment by valentine195** [comment]
You want to increase max zoom.

**Reply by simonfruehauf** [reply]
If I increase max zoom, the picture starts out smaller? I am able to zoom in further yes, but only due to the fact that I start further away from it.

**Reply by simonfruehauf** [reply]
![zoom](https://user-images.githubusercontent.com/25452923/143301826-ab35d3f7-0a93-465c-b0a5-d3487cdc8f12.gif)
This is what I mean, but I'd like to be able to zoom in further on the picture, not start out farther away.

**Reply by valentine195** [reply]
Does it work if you aren’t side by side (as in, only have one open, and switch between edit and preview)? That setup can cause some weirdness to occur. 

But basically, minZoom is how far you can zoom OUT (and can be negative). maxZoom is how far you can zoom IN.

**Reply by simonfruehauf** [reply]
It also doesn't work if I'm side by side. I tried 
maxZoom: 5
minZoom: -1
but that also just resulted in being able to zoom in to a maximum as shown in the gif above...

**Reply by simonfruehauf** [reply]
Still can't seem to figure it out... 

I further played around with minimum and maximum zoom, but can't seem to get any closer than what the gif shows (apart from cropping the image and saving it as a copy in my vault, and then using it as a map).

**Comment by valentine195** [comment, marked as answer]
Hey, sorry I haven't responded in awhile, but I actually figured this out.

Try setting a `bounds` parameter on your map, and then increasing max zoom. It should allow you to zoom in further then.

**Reply by simonfruehauf** [reply]
That just seems to zoom out even further, and make the original image small / move the markers etc.
![fc-bug3](https://user-images.githubusercontent.com/25452923/146408200-bdd538db-b922-4326-8551-df00f0741fd9.gif)

**Reply by valentine195** [reply]
Yes, adding bounds changes the image overlay's coordinates, so unfortunately objects on it will not be defined correctly anymore.

I can confirm that adding bounds that are properly scaled to the image allows the map to be zoomed in further, though

https://user-images.githubusercontent.com/38669521/146410215-42ef637b-66c3-4f67-b7ce-529bdea8db35.mp4

.

**Reply by simonfruehauf** [reply]
I am extremely sorry, I just put in the wrong formatting of the bounds. 

That solves this issue, thank you.

**Reply by valentine195** [reply]
No worries! Glad we could figure it out.
